mycpp/mylib2.cc

```diff
@@ -2,9 +2,9 @@
 
 #include "mylib2.h"
 
-#include <errno.h>   // errno global
 #include <unistd.h>  // isatty
 
+#include "myerror.h"
```

Comment:
Try `grep 'clang-format off' cpp/*.h` and follow the  style there.   "myerror.h" goes at the top so that errno is not defined yet

Also why is the only change in the file?

I think it may solve the problem to put `putenv()` in `posix.cc` instead of `posix.h`.